src/PowerShellEditorServices/Services/PowerShellContext/Session/Host/EditorServicesPSHostUserInterface.cs

```diff
@@ -729,7 +729,7 @@ private async Task WritePromptStringToHostAsync(CancellationToken cancellationTo
             {
             }
 
-            PSCommand promptCommand = new PSCommand().AddScript("prompt");
+            PSCommand promptCommand = new PSCommand().AddScript("prompt", useLocalScope: true);
```

Comment:
If `prompt` is usually dot-sourced in normal PowerShell, it might be worth adding a comment here. If this actually converts us to use normal behaviour, then no comment needed I imagine

Reply:
How silly of me... `prompt` is a function... so I switched to using `AddCommand` :)